help/md/Integer.md

```diff
@@ -1,9 +1,16 @@
 ## name
 Integer
 ## title
+Integer
 ## description
+Integer Datatype
 ## details
+This datatype includes all whole, positive and negative numbers.
 ## authors
 ## see_also
+Natural
+IntegerPos
```

Comment:
Hmm... `IntegerPos` is an interesting data type. Apparently we only use it to specify the `printgen` argument in monitors. Maybe we shouldn't list it here, so that users don't get confused about what the difference between `Natural` and `IntegerPos` is.